src/DEV-1/Randomizer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_1
+{
+    class Randomizer
+    {
+        public static void outputRandomOptions(string[] options, int numberOfNeededOptions)
+        {
+            ArrayList ArrayListOfOptions = new ArrayList(Parser.parseArrayToArrayList(options));
+            string result = "";
+            Random random = new Random();
+            try
+            {
+                for (int i = 0; i < numberOfNeededOptions; i++)
```

Comment:
It will work probabley but too complex and not effective logics, check number of options beforehands. Logic can't use exceptions